alignak/action.py

```diff
@@ -394,6 +395,7 @@ def execute__(self, force_shell=sys.version_info < (2, 7)):
             # http://www.doughellmann.com/PyMOTW/subprocess/ for
             # detail about this.
             try:
+                logger.debug("Launch command: %s", self.command)
```

Comment:
I would suggest to put this line 185 so that you only put once.

```
return self.execute__() # OS specific part
```

command is a attribute, you have access to it at this line